CONTRIBUTING.md

```diff
@@ -53,6 +53,13 @@ Run `make docs` to regenerate documentation for newly added resources and schema
 make docs
 ```
 
+## Signing your Commits
+We require all commits on the contributing PR to be signed.
+
+- [Learn more about signing commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)
+- [Signing old commits](https://stackoverflow.com/questions/41882919/is-there-a-way-to-gpg-sign-all-previous-commits)
```

Comment:
I often manually squash my own commits, in which case it isn't needed, but it can definitely be handy to be able to go back and sign old ones, I've had to do it once after we switched to requiring signatures